packages/documentation/pages/usage/components/inline-edit-2.vue

```diff
@@ -0,0 +1,84 @@
+<template lang="md">
+<ComponentInfo v-bind="{ component }" />
+
+## Basic
+
+<div class="element-example white">
+	<KtInlineEdit2
+		label="Instructions"
+		placeholder="Click to edit"
+		value="my instructions"
+	/>
+</div>
+
+```html
+<KtInlineEdit2
+	label="Instructions"
+	placeholder="Click to edit"
+	value="my instructions"
+/>
+```
+
+## Multiline
+
+<div class="element-example white">
+	<KtInlineEdit2
+		:isMultiLine="true"
+		label="Instructions"
+		placeholder="Edit instructions"
+		value="my instructions"
+	/>
+</div>
+
+```html
+<KtInlineEdit2
+	:isMultiLine="true"
+	label="Instructions"
+	placeholder="Edit instructions"
+	value="my instructions"
+/>
+```
+
+## Usage
+
+### Attributes
+
+| Attribute     | Description                  | Type      | Accepted values | Default           |
+| :------------ | :--------------------------- | :-------- | :-------------- | :---------------- |
+| `isMultiLine` | allow input enter multilines | `Boolean` | —               | `false`           |
+| `label`       | label text                   | `String`  | —               | —                 |
+| `placeholder` | value to display             | `String`  | —               | `"Click to edit"` |
```

Comment:
default should be translatable, let's quickly meet and see how we can integrate a translatable placeholder :)

Reply:
done locally 💯